roles/reproducer/tasks/main.yml

```diff
@@ -142,6 +142,17 @@
       }}
   failed_when: false
 
+- name: Ensure custom networks can be routed outside to the hypervisor
```

Comment:
2 things:

you probably want to ensure firewalld is used, like we do here for instance:
https://github.com/openstack-k8s-operators/ci-framework/blob/52261a3c2e260af2379c6fb0f45120dd204c66b5/roles/libvirt_manager/tasks/create_networks.yml#L84-L88

and you probably should just ensure the created networks/ports are in the `libvirt` zone, like we do here:
https://github.com/openstack-k8s-operators/ci-framework/blob/52261a3c2e260af2379c6fb0f45120dd204c66b5/roles/libvirt_manager/tasks/create_networks.yml#L90-L102

You can loop on the nmstate networks I guess - and reproduce that same behavior.